.template-helpers/variant-windowsservercore.md

```diff
@@ -0,0 +1,8 @@
+## `%%REPO%%:windowsservercore`
+
+This image is based on [Windows Server Core (`microsoft/windowsservercore`)](https://hub.docker.com/r/microsoft/windowsservercore/). As such, it only works in places which that image does, such as Windows 10 (with insider previews) or Windows Server 2016.
+
+For information about how to get "Docker on Windows" installed and configured, please see the relevant "Quick Start" guide provided by Microsoft:
```

Comment:
Hah!  They download Docker directly from https://get.docker.com now instead of preview builds via aka.ms, and they're actually a lot shorter than they appear at a glance (something like 4-5 actual commands to run). :+1: